netlify/functions/manage-findings.ts

```diff
@@ -38,37 +38,29 @@ import { apiKey, domain } from "../_config";
 
 async function getFinding(
   username: string,
-  contest: Contest,
+  repoName: string,
```

Comment:
so I was going to complain that we lost a more semantic type, but frankly the end result here is **more** readable! 🤔

Reply:
Yeah, really good point.

Reply:
Added better type declarations